internal/ct/handlers.go

```diff
@@ -351,13 +351,13 @@ func addChainInternal(ctx context.Context, opts *HandlerOptions, log *log, w htt
 	index, dedupTimeMillis, err := log.storage.Add(ctx, entry)
 	if err != nil {
 		if errors.Is(err, tessera.ErrPushback) {
+			tooManyRequestsAttribute := fmt.Sprintf("tessera_pushback_%s", strings.Split(err.Error(), " ")[0])
```

Comment:
This adds a monitoring-critical dep on the _text_ of the returned error, is there a way we can make this safer?

Reply:
We can add a field to pushback errors in Tessera if you're ok with this?

Reply:
Yeah, I think that could work...